pages/ShoppingCart/index.js

```diff
@@ -0,0 +1,71 @@
+// Import external resources
+import styled from "styled-components";
+
+// Import internal resources
+import Link from "next/link";
+import CartItemCard from "@/components/CartItemCard";
+
+// Import helper functions
+import { formatNumberToDeCurrency } from "@/helpers/formatNumberToCurrency";
+
+export default function ShoppingCartPage({ cartItems }) {
+  console.log(cartItems);
+
+  const sumUpArray = (accumulator, currentValue) => accumulator + currentValue;
+
+  const sumOfAllItemPrices = cartItems
+    .map((item) => item.price * item.amount)
+    .reduce(sumUpArray, 0);
+
+  const sumOfAllItemAmount = cartItems
+    .map((item) => item.amount)
+    .reduce(sumUpArray, 0);
+
+  return (
+    <>
+      <StyledMainContent>
+        <StyledTopInfoSection>
+          <Link href={"/"}>back</Link>
+          <h2>Shopping Cart</h2>
+          <p>{`Items: ${sumOfAllItemAmount}`}</p>
+          <p>{`Cartvalue: ${formatNumberToDeCurrency(sumOfAllItemPrices)}`}</p>
+        </StyledTopInfoSection>
+        <section>
+          <StyledCartItemList>
+            {cartItems.length === 0 ? (
+              <li>No items added</li>
+            ) : (
+              cartItems.map((eachItem) => {
+                return <CartItemCard key={eachItem.id} eachItem={eachItem} />;
+              })
+            )}
+          </StyledCartItemList>
+        </section>
```

Comment:
Do you need the section here? It doesn't seem to have a functionality.

Reply:
Removed the section for now.
Maybe I will add it again if I actually need it later.
- [x] Removed